<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-8M27IT2" failures="1" tests="10" name="tutorialsNinja.LoginTest" time="110.144" errors="0" timestamp="2023-05-30T00:45:35 EDT" skipped="0">
  <testcase classname="tutorialsNinja.LoginTest" name="vrfyUsrNvgtTloginHmPge" time="9.532"/>
  <system-out/>
  <testcase classname="tutorialsNinja.LoginTest" name="verifyTheNumberOfUnsuccessfulLoginAttempts" time="9.672"/>
  <system-out/>
  <testcase classname="tutorialsNinja.LoginTest" name="checkPasswordVisibility" time="10.450">
    <failure message="Password is visible to the page source expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Password is visible to the page source expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at tutorialsNinja.LoginTest.checkPasswordVisibility(LoginTest.java:131)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
]]>
    </failure>
  </testcase> <!-- checkPasswordVisibility -->
  <system-out/>
  <testcase classname="tutorialsNinja.LoginTest" name="testLoginFieldsPlaceholders" time="15.055"/>
  <system-out/>
  <testcase classname="tutorialsNinja.LoginTest" name="verifyForgottenPasswrdLinkDsp" time="9.156"/>
  <system-out/>
  <testcase classname="tutorialsNinja.LoginTest" name="testLoginWithCredentials" time="12.135"/>
  <system-out/>
  <testcase classname="tutorialsNinja.LoginTest" name="testLoginWithCredentials" time="10.965"/>
  <system-out/>
  <testcase classname="tutorialsNinja.LoginTest" name="testLoginWithCredentials" time="13.960"/>
  <system-out/>
  <testcase classname="tutorialsNinja.LoginTest" name="testLoginWithCredentials" time="9.455"/>
  <system-out/>
  <testcase classname="tutorialsNinja.LoginTest" name="testLoginWithCredentials" time="9.764"/>
  <system-out/>
</testsuite> <!-- tutorialsNinja.LoginTest -->
